### ISSUE TITLE
in BO - "allow orders" option in Product Page should override option "hide unavailable attributes" 

### ISSUE THREAD

**Issue by imagicuus** [PrestaShop/PrestaShop#19604]
#### Describe the bug

When in the store settings you want to hide unavailable attributes but for some products you allow orders when they are out of stock there is some weird behaviour. 

#### Expected behavior

In my opinion the "allow orders" product option should override the store option "hide unavailable attributes"

#### Steps to Reproduce

See video on Prestashop Demo site: https://drive.google.com/file/d/1p1sPI1jc8WAojfTml7nubTg3ZFFf1iez/view

#### Additional information

* PrestaShop version: 1.7.6.5
* PHP version: 7

**Comment by florine2623** [comment]
Hi @imagicuus ,
If I understand well, in your video example, you would like to be able to buy the white shirt in S because you allowed it on the product sheet ?

**Comment by imagicuus** [comment]
@florine2623 that is correct

**Comment by florine2623** [comment]
In your video, these 2 options are mentioned :
In Shop Parameters > Product Settings :
1) Display unavailable product attributes on the product page -> **NO**
2) Allow ordering of out-of-stock products -> **YES**

Just to be sure, you would want to have the second option overriding the first one ?

**Comment by imagicuus** [comment]
Yes indeed, because right now it's not hiding properly and you also can't order the option. So even if people would prefer the override the other way around there is still a bug going on.

**Comment by imagicuus** [comment]
@florine2623 all the information needed is given, right? Can you set the issue on To Do?

**Comment by florine2623** [comment]
Hi @PrestaShop/prestashop-product-team ,

@imagicuus would like to see an improvement.
With the following options (in Product Settings), even when a combination product is out-of-stock, if it is chosen to allow the orders in the product sheet, a customer should be able to buy that product.

In BO > Advanced Parameters > Product Settings :
Display unavailable product attributes on the product page -> **NO**
![Capture d’écran 2020-06-12 à 17 11 52](https://user-images.githubusercontent.com/16019289/84517633-d7fb4d80-accf-11ea-8127-f9ff46e715b5.png)

Allow ordering of out-of-stock products -> **YES**
![Capture d’écran 2020-06-12 à 17 12 17](https://user-images.githubusercontent.com/16019289/84517662-e3e70f80-accf-11ea-88ed-b5c14e865347.png)

Now, with a combination product, a product can't be bought even when the default behavior is to Allow orders.

Thanks!

**Comment by colinegin** [comment]
Why don't you choose to display unavailable product attributes if you want to allow orders ?

**Comment by imagicuus** [comment]
@colinegin I've products that are not always on stock at my suppliers. I want those combinations to hide when out of stock. I've also products that are always on stock at my suppliers. People can order those when out of stock at my store.

**Comment by Nukium** [comment]
Hello, I think there is a possible fix here : https://github.com/PrestaShop/PrestaShop/pull/14452

**Comment by vzlatev** [comment]
Hello,

I can confirm the problem and use case is really logical. Low level option "allow order" on specific products should override the higher level general configuration.

**Comment by Hlavtox** [comment]
These options are not related at all. Converting to discussion
